app/javascript/gobierto_data/webapp/components/sets/Nav.vue

```diff
@@ -175,11 +184,17 @@ export default {
   },
   methods: {
     changeTab() {
-      this.activateTab(1);
+      this.activeDatasetTab = 1
     },
-    activateTab(index) {
-      this.activeTab = index;
-      this.$emit("active-tab", index);
+    activateTab(index, label) {
+      this.$emit("active-tab", index)
+      this.$router.push({
+        name: label,
+        params: {
+          id: this.$route.params.id,
+          title: label
```

Comment:
`label` comes from, i.e.`activateTab(0, 'resumen')`. Is this text going to be displayed anywhere? If so, it must be translated. Never use string literals if they're gonna be displayed.